src/month.rs

```diff
@@ -231,8 +231,16 @@ impl Months {
 
 /// An error resulting from reading `<Month>` value with `FromStr`.
 #[derive(Clone, PartialEq, Eq)]
-pub struct ParseMonthError {
-    pub(crate) _dummy: (),
```

Comment:
Why are the `_dummy` fields removed? This makes it possible for library users to construct the type, and we can no longer add fields in the future (not that I think we will...). Can you revert this part?

Reply:
The dummy field is not needed, constructing works by doing `let e = ParseMonthError;`, and why can't you add fields at a later time? I fail to see the issues you raised, sorry, especially since `pub struct ParseWeekdayError {}` is defined the same (which was not added by me).

About running CI, not sure how to do that either, sorry. A push might work, but that implies a further needed commit.

Reply:
A private field is a common future proofing trick. See the somewhat related https://rust-lang.github.io/api-guidelines/future-proofing.html#structs-have-private-fields-c-struct-private. But yes, it is not used consistently at the moment, and not all that interesting for this type.

But I wanted to call it out, as it is an unrelated change.

Reply:
I don't see it as an unrelated change, as it makes it consistent with ParseWeekdayError. And even with your future proofing argument I still find this field unnecessary for this very application. However I can add it back, also to ParseWeekdayError if desired.

Reply:
If you could please add it back and not change `ParseWeekdayError` than this PR will just be simple to approve :smile:.